src/Conference/AddConferenceForm.test.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { Router, MemoryRouter } from "react-router-dom";
+import { act } from "react-dom/test-utils";
+import { mount, configure } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { createMemoryHistory } from "history";
+import { WrappedAddConferenceForm as AddConferenceForm } from "./AddConferenceForm";
+import { createNewConference } from "../api";
+
+configure({ adapter: new Adapter() });
+
+jest.mock("../api", () => ({
```

Comment:
This would only work because the mock is hoisted, but you should be explicit about this being _before_ the import api (rather than importing the original and then mocking future imports)